plugins/cameraserver/src/main/resources/edu/wpi/first/shuffleboard/plugin/cameraserver/widget/CameraServerWidget.fxml

```diff
@@ -14,6 +14,7 @@
 <?import javafx.scene.layout.GridPane?>
 <?import javafx.scene.layout.Pane?>
 <?import javafx.scene.control.Button?>
+<?import com.jfoenix.controls.JFXSlider?>
```

Comment:
You have a now unused import a few lines up: `<?import javafx.scene.control.Slider?>`